client/src/components/IndexPage.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { Link } from 'react-router';
+import AuthHeader from './headers/AuthHeader';
+
+/**
+ * IndexPage class declaration
+ */
+export default class IndexPage extends React.Component {
```

Comment:
Since you are not using any class method here, you should make this and others like this a stateless component